lib/flowDiagnostics.js

```diff
@@ -110,7 +110,7 @@ async function requestDiagnostics(context: ExtensionContext, document: TextDocum
   coverage.update(document.uri);
 
   if (pendingDiagnostics.get(id) === version) {
-    pendingDiagnostics.delete(id);
+    pendingDiagnostics.delete(id); getFileExtensions
```

Comment:
is this line a mistake?

Reply:
100% a mistake... must have been a mistaken paste - holy cow!